Dockerfile

```diff
@@ -51,7 +51,7 @@ COPY static /app/static/
 COPY src/styles /app/src/styles/
 
 ENV ROCKET_PORT 80
-ENV ROCKET_ENV prod
+ENV ROCKET_PROFILE release
```

Comment:
From the [changelog](https://github.com/SergioBenitez/Rocket/blob/master/CHANGELOG.md#silent-changes):

> ROCKET_ENV is now ROCKET_PROFILE. A warning is emitted a launch time if the former is set.